catalog/confluence/catalog-confluence-source/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -62,7 +62,7 @@
     </OCD>
 
 
-    <OCD description="Confluence Connected Source" name="Confluence Connected Source"
+    <OCD description="Connected Source interface for retrieving pages, comments, and attachments from an Atlassian Confluence® Server" name="Confluence Connected Source"
```

Comment:
❓ Is it worth describing the difference between federated or connected in these descriptions? 
I think this is fine as is, just a thought.

Reply:
I would say it isn't since we have a section in the docs that describes federation.